app/assets/javascripts/index/search.js

```diff
@@ -18,9 +21,14 @@ OSM.Search = function (map) {
   $(".search_form").on("submit", function (e) {
     e.preventDefault();
     $("header").addClass("closed");
-    const query = $(this).find("input[name=query]").val();
-    let search = "/";
-    if (query) search = "/search?" + new URLSearchParams({ query });
+    const params = new URLSearchParams;
+    for (const paramName of ["query", "zoom", "minlon", "minlat", "maxlon", "maxlat"]) {
```

Comment:
If we're going to build the URL in the submit handler why not just add the parameters from the bounding box here instead of adding hidden input fields and an event handler to keep them up to date?

Reply:
That was my attempt to remove `submit` listener. I thought about letting Turbo handling the entire thing. But since I didn't complete that, I'll get the zoom and bbox here.